extensions/positron-python/src/client/interpreter/display/index.ts

```diff
@@ -133,7 +133,9 @@ export class InterpreterDisplay implements IInterpreterDisplay, IExtensionSingle
         if (this.statusBar) {
             if (interpreter) {
                 this.statusBar.color = '';
-                this.statusBar.tooltip = this.pathUtils.getDisplayName(interpreter.path, workspaceFolder?.fsPath);
+                // --- Start Positron --
+                this.statusBar.tooltip = 'Select Interpreter';
```

Comment:
Is there a constant used by the corresponding command that we could use here?

Reply:
Swapped for the localized title from the Interpreter QuickPick!